Manual/VCAL_MANUAL_for_PREPROCESSING_IFS.txt

```diff
@@ -52,9 +52,9 @@ badfr_crit_names_psf = ['corr'] # Bad frame criteria to be used for PSF cubes; s
 	- "crop_sz": in fwhm, cfr. description of vip_hci.preproc.cube_detect_badfr_ellipticity(). Default: 10 (for 10*fwhm crop)
 # corr: rej based on correlation level with a frame or median frame. Possible dico entries: 
 	- "thr": min correlation level with median frame. Notes: supersedes perc if both are provided; use "plot" to check a plot of correlation level for the frames of the cube.
-	- "perc": min percentile for bad frames (e.g. if set to 10, the 10% worst frames in the cube will be automatically rejected) . Default: 0
+	- "perc": min percentile for bad frames (e.g. if set to 10, the 10% worst frames in the cube will be automatically rejected) . Default: 10
```

Comment:
I'd leave it to 0 as default value. For data taken in very good conditions, no need to waste photons by discarding good frames. Ideally, one wants to check the cubes without discarding frames first, before re-running this step with an appropriate value for perc if needed.